python/python/glide/async_commands/server_modules/json.py

```diff
@@ -147,6 +147,52 @@ async def get(
     return cast(TJsonResponse[Optional[bytes]], await client.custom_command(args))
 
 
+async def arrappend(
+    client: TGlideClient,
+    key: TEncodable,
+    path: TEncodable,
```

Comment:
why isnt the path an optional per docs?

Reply:
optional in redis, not aws, the decision was to stick with aws implementation

Reply:
Can you explain that?
If I'm using json with a regular Valkey-server with json, not cloud provided server, is it optional?

Reply:
As I see it, optional. It doesn't block AWS users, they just need to have it documented to know that for them, it's a must.
We would rather not block other user types. Tomorrow we might have some user open an issue on it.
If there's a real conflict between implementations, we can discuss with what to stick or create two APIs for the different cases.
But we would rather support all cases when those are possible together with no real issue.
Misuse is always possible, like users with a lower version using the API available in a higher version. We don't block all users from using the API, since we have some version preferences.
We can consider more sophisticated ways if you think it's not perfect, but it requires some thinking.

Reply:
In particular, the usage of this command without path is quite a narrow case: it indicates that the entire JSON is just an array at its root. I would say this is, to some degree, the justification of AWS JSON module making path mandatory.